mariadb-10.11.advisories.yaml

```diff
@@ -76,3 +76,8 @@ advisories:
             componentType: java-archive
             componentLocation: /usr/mysql-test/plugin/connect/connect/std_data/JdbcMariaDB.jar
             scanner: grype
+      - timestamp: 2024-06-25T18:11:51Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-in-execution-path
+          note: This CVE appears to stem from a hardcoded postgres jar file used specifically for a stale test
```

Comment:
I'd like to get confirmation that this is still the case with the recent related package PR — and if not we can update this type and note